spec/units.yaml

```diff
@@ -214,19 +214,6 @@ years:
   quantity: duration
   allowed-datatypes: ['numeric']
 
-# Datetime
-
-unix-time:
-  definition: Number of non-leap seconds which have passed since 00:00:00 UTC on Thursday, 1 January 1970
-  unit: UNIX Timestamp
-  quantity: datetime
```

Comment:
We need to discuss if we still want to use something like `quantity` for formats, to indicate that some formats represent the same type of content, like if you use `iso8601` you could possibly convert it to `unix-time-ms`, but not to `ipv4`.

If we do not want to reuse the quantity concept then we should possibly remove `datetime` from the quantity file.

Reply:
That's a good point. I did not think about quantities so much here